crates/wasmtime/src/store.rs

```diff
@@ -79,14 +157,50 @@ pub struct Store<T> {
     inner: ManuallyDrop<Box<StoreInner<T>>>,
 }
 
+/// Internal contents of a `Store<T>` that live on the heap.
+///
+/// The members of this struct are those that need to be generic over `T`, the
+/// store's internal type storage. Otherwise all things that don't rely on `T`
+/// should go into `StoreOpaque`.
 pub struct StoreInner<T> {
```

Comment:
Can this be `pub(crate)`?

Reply:
Er forgot to answer this before I merged, but I don't think so. There's a lot of `pub` stuff already and I frequently get compiler errors about pub-crate things being in pub-interfaces, so I prefer to not deal with any of that.